primer.md

```diff
@@ -153,8 +154,18 @@ Extension attributes to the CloudEvent specification are meant to
 be additional metadata that needs to be included to help ensure proper
 routing and processing of the CloudEvent. Additional metadata for other
 purposes, that is related to the event itself and not needed in the
-transportation of the CloudEvent, should instead be placed within the proper
-extensibility points of the event itself.
+transportation or processing of the CloudEvent, should instead be placed
+within the proper extensibility points of the event (the `data` attribute)
+itself.
+
+The specification places no restrictions on the type of the extension
```

Comment:
Metadata in the format of property/classification bag is not exactly the same as structured data format since we can only define the name of the bag and format of the bag, but have to leave the key-value pairs inside the bag to be customizable and added by the event producer. Suggest to reword the text as follows to avoid confusion

Meaning, they may be simple types (e.g. string, integers), or structured data, or property bags. 

This applies to metadata defined in both the main spec and the extensions spec.

Reply:
I added some text for this.